src/server/sql/reading/create_function_get_compare_readings.sql

```diff
@@ -57,40 +59,32 @@ DECLARE
 BEGIN
 	curr_tsrange := tsrange(curr_start, curr_end);
 	prev_tsrange := tsrange(curr_start - shift, curr_end - shift);
-
+	-- Modified to retrieve converted hourly readings from the materialized view.
 	RETURN QUERY
 	WITH
 	curr_period AS (
 		SELECT
-			meters.id AS meter_id,
-			-- Convert the reading based on the conversion found below.
-			-- It is okay to sum the flow units because hourly readings has a rate of per hour so
-			-- to convert to a quantity would multiply by 1 so it is the same formula.
-			SUM(hourly.reading_rate) * c.slope + c.intercept AS reading
-		FROM (((hourly_readings_unit hourly
-		INNER JOIN unnest(meter_ids) meters(id) ON hourly.meter_id = meters.id)
-		INNER JOIN meters m ON m.id = meters.id)
-		-- This is getting the conversion for the meter and unit to graph.
-		-- The slope and intercept are used above the transform the reading to the desired unit.
-		INNER JOIN cik c on c.source_id = m.unit_id AND c.destination_id = graphic_unit_id)
-		WHERE curr_tsrange @> hourly.time_interval
-		GROUP BY meters.id, c.slope, c.intercept
+			hourly.meter_id AS meter_id,
+			SUM(hourly.reading_rate) AS reading
```

Comment:
I think it might be useful to put back the second part of the removed comment on flow unit.